backend/main-service/src/main/java/kkakka/mainservice/category/ui/dto/ResponseCategoryProducts.java

```diff
@@ -0,0 +1,17 @@
+package kkakka.mainservice.category.ui.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class ResponseCategoryProducts {
+    private Long id;
+    private String name;
+    private Integer price;
+    private String image_url;
+    private Integer discount;
+    private Date registered_at;
```

Comment:
```suggestion
    private String imageUrl;
    private Integer discount;
    private Date registeredAt;
```

자바에서는 카멜케이스로 쓰는게 어떨까요!